gulpfile.js

```diff
@@ -522,7 +520,7 @@ gulp.task('cordova-build', ['cordova-create'], function (callback) {
 gulp.task('cordova-run', function (callback) {
 	var platform = args.ios ? 'ios' : 'android';
 
-	exec('(cd ./.cordova/ && cordova run ' + platform + ')', function (err, stdout) {
+	process.exec('(cd ./.cordova/ && cordova run ' + platform + ')', function (err, stdout) {
```

Comment:
May as well delete `exec = require('child_process').exec,` from the top if you're changing this